api/query_engine_test.go

```diff
@@ -36,6 +37,7 @@ func testPlan(reqs []models.Req, retentions []conf.Retentions, outReqs []models.
 			maxPointsPerReqSoft = points
 		}
 	}
+	maxPointsPerReqHard = maxPointsPerReqSoft * 10
```

Comment:
why do we need this? was it rejecting requests?

Reply:
It was failing during planning because it was always set to 0

https://github.com/grafana/metrictank/blob/244433ae45f46cc9188df52456565b2bd03e6763/api/query_engine.go#L144-L147

Reply:
I see, but 0 means disabled. I will update the code to implement 0/disabled. then we can probably undo that change

Reply:
That makes more sense, sounds good.